packages/components/src/__future__/Tag/Tag.module.scss

```diff
@@ -0,0 +1,66 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/typography";
+
+.container {
+  display: inline-flex;
+  align-items: center;
+  padding: 5px 0.5rem 5px 0.5rem;
+  gap: 0.25rem;
+  border-radius: 28px;
+  font-family: $typography-paragraph-intro-lede-font-family;
+  font-size: $typography-paragraph-small-font-size;
+  font-weight: $typography-paragraph-small-font-weight;
+  line-height: $typography-paragraph-small-line-height;
+  white-space: nowrap;
+}
+
+.gray {
+  background-color: $color-gray-200;
+  color: $color-purple-800;
+}
+.blue {
+  background-color: $color-blue-100;
+  color: $color-blue-700;
+  svg {
+    color: $color-blue-500;
+  }
```

Comment:
I think this would affect the icon in the dismiss button, right? Might be best to separate the icon styles.